LM IR BLinker.X/main.c

```diff
@@ -94,6 +101,14 @@ void main(void)
        
     }
 }
-/**
- End of File
-*/
\ No newline at end of file
+
+
+// ========== LOCAL FUNCTIONS ==========
+void bitTimerInterrupt() {
+    PWM_EN = pattern[patternIndex];

```

Comment:
A "C style" array. You can access an element of an array by calling out `variable[index]`. Don't forget that arrays start with element 0 (`variable[0], variable[1],..., variable[size-1]`)